### ISSUE TITLE
Unable to distinguish between unsigned PSBTs on coldcard

### ISSUE THREAD

**Issue by billygarrison** [jamesob/coldcore#33]
The coldcard display truncates all of the following transactions:
```
unsigned-20210307-1910.psbt
unsigned-20210307-1935.psbt
unsigned-20210307-1938.psbt
```

Which makes them all look the same:
```
unsigned-20210307
unsigned-20210307
unsigned-20210307
```

Not sure if the naming format is required for PSBTs, but just wanted to bring this to your attention in case it is an easy fix.

**Comment by jamesob** [comment]
Good point - I'll look at renaming the unsigned PSBTs we generate to something that doesn't get stripped.
